src/actions/request.rs

```diff
@@ -121,57 +120,47 @@ impl Request {
       interpolated_url
     };
 
-    let client = if interpolated_base_url.starts_with("https") {
-      // Build a TSL connector
-      let mut connector_builder = TlsConnector::builder();
-      connector_builder.danger_accept_invalid_certs(config.no_check_certificate);
-
-      let ssl = NativeTlsClient::from(connector_builder.build().unwrap());
-      let connector = HttpsConnector::new(ssl);
-
-      Client::with_connector(connector)
-    } else {
-      Client::new()
-    };
-
+    // let client = ClientBuilder::default().danger_accept_invalid_certs(config.no_check_certificate).build().unwrap();
```

Comment:
This needs a little refactor to pass `no_check_certificate` config to `reqwest::Client` in `Request::new`

Reply:
Don't worry about it for now. After merging this async branch I want to start dealing with the keep-alive feature. It is a requirement for a long time that will increase the throughput drastically.

The idea is to have a pool of clients, (domain -> Client), to be used here. The domain will be determined from the `interpolated_url`.

Reply:
I created this branch forked this one to play a bit. https://github.com/fcsonline/drill/commit/e6b51ffbfecf68f9f5265d54c74e1091a2e10599

Keep Alive is awesome! :muscle: